react/redux/redux/src/app/screens/App/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component, Fragment } from 'react';
+
+import store from '~redux/store';
+
+import Navbar from '~components/Navbar';
+
+import Footer from '~components/Footer';
+
+import actionsCreators from '~redux/book/actions';
+
+import Book from './components/Book';
+import Search from './components/Search';
+import ShoppingCart from './components/ShoppingCart';
+import styles from './styles.scss';
+
+class App extends Component {
+  state = {
+    books: [],
+    bookSelected: []
+  };
+
+  componentDidMount() {
+    store.subscribe(() => {
+      const { books, bookSelected } = store.getState();
+      this.setState({ books, bookSelected });
+    });
```

Comment:
It will be great to let the trainees translate this to `mapStateToProps / mapDispatchToProps` 😊